src/users/users.service.spec.ts

```diff
@@ -568,9 +565,16 @@ describe('UsersService', () => {
       // Because secondUser caught a bug first, we consider secondUser to be
       // ranked earlier than firstUser. The last event for secondUser doesn't
       // count because it has 0 points.
```

Comment:
Didn't want to move these tests because I think we will run into a cyclic dependency issue, would have to move the whole controller